AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -30,6 +30,7 @@ struct SessionHeaderView: View {
                 Spacer()
                 if session.type == .fixed {
                     actionsMenuFixed
+                        .disabled(true)
```

Comment:
maybe it would be better to remove this actionsMenuFixed at all for the time being?

Reply:
I really like this idea - done